IDEAS/Fluid/Taps/TapProfile.mo

```diff
@@ -0,0 +1,84 @@
+within IDEAS.Fluid.Taps;
+model TapProfile
+  "Model for a DHW tap, reading the DHW demand profile via a CombiTimeTable."
+  extends IDEAS.Fluid.Taps.BaseClasses.PartialTap;
+
+  parameter Real uniCon=1/60
+    "Conversion factor to convert units from DHW profile [l/min] to [kg/s]";
+  parameter String loadFile=Modelica.Utilities.Files.loadResource(
+    "modelica://IDEAS/Resources/domestichotwaterprofiles/DHW_1year_2adults.txt") annotation(Dialog(loadSelector(filter="All files (*.*)", caption="Select the DHW profile file")));
+
+  Modelica.Blocks.Sources.CombiTimeTable pro(
+    tableOnFile=true,
+    tableName="data",
+    fileName=loadFile,
+    smoothness=Modelica.Blocks.Types.Smoothness.LinearSegments,
+    extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic,
+    timeScale=1,
+    timeEvents=Modelica.Blocks.Types.TimeEvents.NoTimeEvents)
+    "DHW demand profile input ([l/min] by default)"
+    annotation (Placement(transformation(extent={{-90,-90},{-70,-70}})));
+  Modelica.Blocks.Math.Gain con(k=uniCon)
+    "Conversion from l/min to kg/s using density = 1000 kg/m3" annotation (
+      Placement(visible=true, transformation(extent={{-50,-90},{-30,-70}},
+          rotation=0)));
+
+equation
+  con.y = m_flow_set;
```

Comment:
Reverse the order: `m_flow_set=con.y;`

Reply:
https://github.com/open-ideas/IDEAS/pull/1355/commits/b61fd7ac29a47478ccd67d5f471f69effd447f8c